pcsx2/DEV9/Linux/dev9.ui

```diff
@@ -5,7 +5,7 @@
   <object class="GtkDialog" id="IDD_CONFDLG">
     <property name="can_focus">False</property>
     <property name="border_width">5</property>
-    <property name="title" translatable="yes">HDD Configure</property>
+    <property name="title" translatable="yes">DEV9 Settings</property>
```

Comment:
This may make sense from a developer point of view but it's user unfriendly. Nobody cares about the internal chip's name

Reply:
I went with that for consistency but yeah it's a bit unfriendly for normal users.
We can adjust the others with better naming as well
SPU2 Settings -> Audio Settings
GSdx -> Video Settings

Reply:
I went ahead with the suggestion and also renamed spu2 since it's been merged in to the core. Can be changed after merge.